Vage foutmelding/stacktrace als topnl datasource niet beschikbaar is bij transformeren BGT

### Probleem
Als de topnl databron miet beschikbaar/onjuist geconfigureerd is en je wil BGTlight transformeren dan treed er een fout op die moeilijk te traceren is... (afgezien van de eerste regel van de foutmelding)

Het zou mooi zijn als deze op het scherm te zien zou zijn ipv de hele stack trace

### Versie
versie info: <!-- ga naar http://<server:poort>/brmo-service/about.jsp en kopier/plak -->1.6.3

### Server log
<details><summary>server log:</summary>

```

2019-09-27 14:35:27 INFO  nl.b3p.brmo.service.scanner.LaadprocesTransformatieUitvoeren#execute - Het laadproces transformatie proces met ID 23 is gestart op Fri Sep 27 14:35:27 CEST 2019.
2019-09-27 14:35:27 ERROR nl.b3p.brmo.service.util.ConfigUtil#getDataSourceTopNL - Fout verbinden naar 'topnl' schema.
javax.naming.NameNotFoundException: Name [rsgbbgttopnl] is not bound in this Context. Unable to find [rsgbbgttopnl].
	at org.apache.naming.NamingContext.lookup(NamingContext.java:817)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:159)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:827)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:159)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:827)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:173)
	at org.apache.naming.factory.ResourceLinkFactory.getObjectInstance(ResourceLinkFactory.java:152)
	at javax.naming.spi.NamingManager.getObjectInstance(NamingManager.java:321)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:840)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:159)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:827)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:159)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:827)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:173)
	at nl.b3p.brmo.service.util.ConfigUtil.getDataSourceTopNL(ConfigUtil.java:107)
	at nl.b3p.brmo.service.scanner.LaadprocesTransformatieUitvoeren.execute(LaadprocesTransformatieUitvoeren.java:106)
	at nl.b3p.brmo.service.stripes.LaadprocesTransformatieUitvoerActionBean.execute(LaadprocesTransformatieUitvoerActionBean.java:80)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.sourceforge.stripes.controller.DispatcherHelper$6.intercept(DispatcherHelper.java:456)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:176)
	at net.sourceforge.stripes.controller.BeforeAfterMethodInterceptor.intercept(BeforeAfterMethodInterceptor.java:113)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at org.stripesstuff.plugin.waitpage.WaitPageInterceptor.intercept(WaitPageInterceptor.java:125)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at net.sourceforge.stripes.controller.ExecutionContext.wrap(ExecutionContext.java:86)
	at net.sourceforge.stripes.controller.DispatcherHelper.invokeEventHandler(DispatcherHelper.java:454)
	at net.sourceforge.stripes.controller.DispatcherServlet.invokeEventHandler(DispatcherServlet.java:278)
	at net.sourceforge.stripes.controller.DispatcherServlet.service(DispatcherServlet.java:160)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at net.sourceforge.stripes.controller.StripesFilter.doFilter(StripesFilter.java:260)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:200)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:607)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:668)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.ajp.AjpProcessor.service(AjpProcessor.java:394)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:834)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1415)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2019-09-27 14:35:27 ERROR nl.b3p.brmo.service.scanner.LaadprocesTransformatieUitvoeren#execute - Fout bij transformeren laadprocessen naar RSGB
nl.b3p.brmo.loader.util.BrmoException: Fout verbinden naar 'topnl' schema.
	at nl.b3p.brmo.service.util.ConfigUtil.getDataSourceTopNL(ConfigUtil.java:111)
	at nl.b3p.brmo.service.scanner.LaadprocesTransformatieUitvoeren.execute(LaadprocesTransformatieUitvoeren.java:106)
	at nl.b3p.brmo.service.stripes.LaadprocesTransformatieUitvoerActionBean.execute(LaadprocesTransformatieUitvoerActionBean.java:80)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.sourceforge.stripes.controller.DispatcherHelper$6.intercept(DispatcherHelper.java:456)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:176)
	at net.sourceforge.stripes.controller.BeforeAfterMethodInterceptor.intercept(BeforeAfterMethodInterceptor.java:113)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at org.stripesstuff.plugin.waitpage.WaitPageInterceptor.intercept(WaitPageInterceptor.java:125)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at net.sourceforge.stripes.controller.ExecutionContext.wrap(ExecutionContext.java:86)
	at net.sourceforge.stripes.controller.DispatcherHelper.invokeEventHandler(DispatcherHelper.java:454)
	at net.sourceforge.stripes.controller.DispatcherServlet.invokeEventHandler(DispatcherServlet.java:278)
	at net.sourceforge.stripes.controller.DispatcherServlet.service(DispatcherServlet.java:160)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at net.sourceforge.stripes.controller.StripesFilter.doFilter(StripesFilter.java:260)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:200)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:607)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:668)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.ajp.AjpProcessor.service(AjpProcessor.java:394)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:834)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1415)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.naming.NameNotFoundException: Name [rsgbbgttopnl] is not bound in this Context. Unable to find [rsgbbgttopnl].
	at org.apache.naming.NamingContext.lookup(NamingContext.java:817)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:159)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:827)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:159)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:827)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:173)
	at org.apache.naming.factory.ResourceLinkFactory.getObjectInstance(ResourceLinkFactory.java:152)
	at javax.naming.spi.NamingManager.getObjectInstance(NamingManager.java:321)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:840)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:159)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:827)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:159)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:827)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:173)
	at nl.b3p.brmo.service.util.ConfigUtil.getDataSourceTopNL(ConfigUtil.java:107)
	... 42 more
2019-09-27 14:35:28 ERROR net.sourceforge.stripes.tag.layout.LayoutComponentTag#error - Unhandled exception trying to render component "contents" to a string in context /WEB-INF/jsp/beheer/laadprocestransformerenuitvoeren.jsp -> /WEB-INF/jsp/layout/default.jsp
org.apache.jasper.JasperException: javax.el.ELException: Error reading [soort] on type [nl.b3p.brmo.persistence.staging.LaadprocesTransformatieProces]
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:638)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:514)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516)
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:868)
	at org.apache.jasper.runtime.PageContextImpl.include(PageContextImpl.java:496)
	at net.sourceforge.stripes.tag.layout.LayoutContext.doInclude(LayoutContext.java:203)
	at net.sourceforge.stripes.tag.layout.LayoutComponentRenderer.write(LayoutComponentRenderer.java:137)
	at net.sourceforge.stripes.tag.layout.LayoutComponentTag.doStartTag(LayoutComponentTag.java:181)
	at org.apache.jsp.WEB_002dINF.jsp.layout.default_jsp._jspx_meth_stripes_005flayout_002dcomponent_005f2(default_jsp.java:358)
	at org.apache.jsp.WEB_002dINF.jsp.layout.default_jsp._jspx_meth_stripes_005flayout_002ddefinition_005f0(default_jsp.java:276)
	at org.apache.jsp.WEB_002dINF.jsp.layout.default_jsp._jspService(default_jsp.java:156)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:476)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516)
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:868)
	at org.apache.jasper.runtime.PageContextImpl.include(PageContextImpl.java:496)
	at net.sourceforge.stripes.tag.layout.LayoutContext.doInclude(LayoutContext.java:203)
	at net.sourceforge.stripes.tag.layout.LayoutRenderTag.doEndTag(LayoutRenderTag.java:175)
	at org.apache.jsp.WEB_002dINF.jsp.beheer.laadprocestransformerenuitvoeren_jsp._jspService(laadprocestransformerenuitvoeren_jsp.java:309)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:476)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712)
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:459)
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:384)
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:312)
	at net.sourceforge.stripes.action.ForwardResolution.execute(ForwardResolution.java:131)
	at net.sourceforge.stripes.controller.DispatcherHelper$7.intercept(DispatcherHelper.java:497)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:176)
	at net.sourceforge.stripes.controller.HttpCacheInterceptor.intercept(HttpCacheInterceptor.java:103)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at net.sourceforge.stripes.controller.BeforeAfterMethodInterceptor.intercept(BeforeAfterMethodInterceptor.java:113)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at org.stripesstuff.plugin.waitpage.WaitPageInterceptor.intercept(WaitPageInterceptor.java:175)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at net.sourceforge.stripes.controller.ExecutionContext.wrap(ExecutionContext.java:86)
	at net.sourceforge.stripes.controller.DispatcherHelper.executeResolution(DispatcherHelper.java:491)
	at net.sourceforge.stripes.controller.DispatcherServlet.executeResolution(DispatcherServlet.java:286)
	at net.sourceforge.stripes.controller.DispatcherServlet.service(DispatcherServlet.java:170)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at net.sourceforge.stripes.controller.StripesFilter.doFilter(StripesFilter.java:260)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:200)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:607)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:668)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.ajp.AjpProcessor.service(AjpProcessor.java:394)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:834)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1415)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.el.ELException: Error reading [soort] on type [nl.b3p.brmo.persistence.staging.LaadprocesTransformatieProces]
	at javax.el.BeanELResolver.getValue(BeanELResolver.java:98)
	at org.apache.jasper.el.JasperELResolver.getValue(JasperELResolver.java:113)
	at org.apache.el.parser.AstValue.getValue(AstValue.java:169)
	at org.apache.el.ValueExpressionImpl.getValue(ValueExpressionImpl.java:190)
	at org.apache.jasper.runtime.PageContextImpl.proprietaryEvaluate(PageContextImpl.java:701)
	at org.apache.jsp.WEB_002dINF.jsp.beheer.laadprocestransformerenuitvoeren_jsp._jspService(laadprocestransformerenuitvoeren_jsp.java:216)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:476)
	... 86 more
Caused by: org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: nl.b3p.brmo.persistence.staging.AutomatischProces.config, no session or session was closed
	at org.hibernate.collection.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:383)
	at org.hibernate.collection.AbstractPersistentCollection.throwLazyInitializationExceptionIfNotConnected(AbstractPersistentCollection.java:375)
	at org.hibernate.collection.AbstractPersistentCollection.readElementByIndex(AbstractPersistentCollection.java:176)
	at org.hibernate.collection.PersistentMap.get(PersistentMap.java:169)
	at nl.b3p.brmo.persistence.staging.LaadprocesTransformatieProces.getSoort(LaadprocesTransformatieProces.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at javax.el.BeanELResolver.getValue(BeanELResolver.java:94)
	... 94 more
2019-09-27 14:35:28 WARN  net.sourceforge.stripes.exception.DefaultExceptionHandler#warn - Unhandled exception caught by the Stripes default exception handler.
org.apache.jasper.JasperException: javax.servlet.ServletException: net.sourceforge.stripes.exception.StripesJspException: An exception was raised while invoking a layout. The layout used was '/WEB-INF/jsp/layout/default.jsp'. The following information was supplied to the render tag: LayoutContext{component names=[contents, html_head, title], parameters={}}
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:638)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:499)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712)
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:459)
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:384)
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:312)
	at net.sourceforge.stripes.action.ForwardResolution.execute(ForwardResolution.java:131)
	at net.sourceforge.stripes.controller.DispatcherHelper$7.intercept(DispatcherHelper.java:497)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:176)
	at net.sourceforge.stripes.controller.HttpCacheInterceptor.intercept(HttpCacheInterceptor.java:103)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at net.sourceforge.stripes.controller.BeforeAfterMethodInterceptor.intercept(BeforeAfterMethodInterceptor.java:113)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at org.stripesstuff.plugin.waitpage.WaitPageInterceptor.intercept(WaitPageInterceptor.java:175)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at net.sourceforge.stripes.controller.ExecutionContext.wrap(ExecutionContext.java:86)
	at net.sourceforge.stripes.controller.DispatcherHelper.executeResolution(DispatcherHelper.java:491)
	at net.sourceforge.stripes.controller.DispatcherServlet.executeResolution(DispatcherServlet.java:286)
	at net.sourceforge.stripes.controller.DispatcherServlet.service(DispatcherServlet.java:170)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at net.sourceforge.stripes.controller.StripesFilter.doFilter(StripesFilter.java:260)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:200)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:607)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:668)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.ajp.AjpProcessor.service(AjpProcessor.java:394)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:834)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1415)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.servlet.ServletException: net.sourceforge.stripes.exception.StripesJspException: An exception was raised while invoking a layout. The layout used was '/WEB-INF/jsp/layout/default.jsp'. The following information was supplied to the render tag: LayoutContext{component names=[contents, html_head, title], parameters={}}
	at org.apache.jasper.runtime.PageContextImpl.handlePageException(PageContextImpl.java:663)
	at org.apache.jsp.WEB_002dINF.jsp.beheer.laadprocestransformerenuitvoeren_jsp._jspService(laadprocestransformerenuitvoeren_jsp.java:328)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:476)
	... 51 more
Caused by: org.apache.jasper.JasperException: An exception occurred processing [WEB-INF/jsp/layout/default.jsp] at line [21]

18:                 <jsp:include page="/WEB-INF/jsp/common/menu.jsp"/>
19:             </div>
20:             <div class="content">
21:                 <stripes:layout-component name="contents"/>
22:             </div>
23:             <div class="footer"></div>
24:         </body>


Stacktrace:
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:625)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:499)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516)
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:868)
	at org.apache.jasper.runtime.PageContextImpl.include(PageContextImpl.java:496)
	at net.sourceforge.stripes.tag.layout.LayoutContext.doInclude(LayoutContext.java:203)
	at net.sourceforge.stripes.tag.layout.LayoutRenderTag.doEndTag(LayoutRenderTag.java:175)
	at org.apache.jsp.WEB_002dINF.jsp.beheer.laadprocestransformerenuitvoeren_jsp._jspService(laadprocestransformerenuitvoeren_jsp.java:309)
	... 54 more
Caused by: javax.servlet.ServletException: net.sourceforge.stripes.exception.StripesJspException: org.apache.jasper.JasperException: javax.el.ELException: Error reading [soort] on type [nl.b3p.brmo.persistence.staging.LaadprocesTransformatieProces]
	at org.apache.jasper.runtime.PageContextImpl.handlePageException(PageContextImpl.java:663)
	at org.apache.jsp.WEB_002dINF.jsp.layout.default_jsp._jspService(default_jsp.java:169)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:476)
	... 67 more
Caused by: org.apache.jasper.JasperException: javax.el.ELException: Error reading [soort] on type [nl.b3p.brmo.persistence.staging.LaadprocesTransformatieProces]
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:638)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:514)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516)
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:868)
	at org.apache.jasper.runtime.PageContextImpl.include(PageContextImpl.java:496)
	at net.sourceforge.stripes.tag.layout.LayoutContext.doInclude(LayoutContext.java:203)
	at net.sourceforge.stripes.tag.layout.LayoutComponentRenderer.write(LayoutComponentRenderer.java:137)
	at net.sourceforge.stripes.tag.layout.LayoutComponentTag.doStartTag(LayoutComponentTag.java:181)
	at org.apache.jsp.WEB_002dINF.jsp.layout.default_jsp._jspx_meth_stripes_005flayout_002dcomponent_005f2(default_jsp.java:358)
	at org.apache.jsp.WEB_002dINF.jsp.layout.default_jsp._jspx_meth_stripes_005flayout_002ddefinition_005f0(default_jsp.java:276)
	at org.apache.jsp.WEB_002dINF.jsp.layout.default_jsp._jspService(default_jsp.java:156)
	... 70 more
Caused by: javax.el.ELException: Error reading [soort] on type [nl.b3p.brmo.persistence.staging.LaadprocesTransformatieProces]
	at javax.el.BeanELResolver.getValue(BeanELResolver.java:98)
	at org.apache.jasper.el.JasperELResolver.getValue(JasperELResolver.java:113)
	at org.apache.el.parser.AstValue.getValue(AstValue.java:169)
	at org.apache.el.ValueExpressionImpl.getValue(ValueExpressionImpl.java:190)
	at org.apache.jasper.runtime.PageContextImpl.proprietaryEvaluate(PageContextImpl.java:701)
	at org.apache.jsp.WEB_002dINF.jsp.beheer.laadprocestransformerenuitvoeren_jsp._jspService(laadprocestransformerenuitvoeren_jsp.java:216)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:476)
	... 86 more
Caused by: org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: nl.b3p.brmo.persistence.staging.AutomatischProces.config, no session or session was closed
	at org.hibernate.collection.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:383)
	at org.hibernate.collection.AbstractPersistentCollection.throwLazyInitializationExceptionIfNotConnected(AbstractPersistentCollection.java:375)
	at org.hibernate.collection.AbstractPersistentCollection.readElementByIndex(AbstractPersistentCollection.java:176)
	at org.hibernate.collection.PersistentMap.get(PersistentMap.java:169)
	at nl.b3p.brmo.persistence.staging.LaadprocesTransformatieProces.getSoort(LaadprocesTransformatieProces.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at javax.el.BeanELResolver.getValue(BeanELResolver.java:94)
	... 94 more


```

</details>

--- a/brmo-service/src/main/java/nl/b3p/brmo/service/scanner/LaadprocesTransformatieUitvoeren.java
+++ b/brmo-service/src/main/java/nl/b3p/brmo/service/scanner/LaadprocesTransformatieUitvoeren.java
@@ -102,8 +102,14 @@ public class LaadprocesTransformatieUitvoeren extends AbstractExecutableProces {
         BrmoFramework brmo = null;
         try {
             DataSource dataSourceStaging = ConfigUtil.getDataSourceStaging();
-            DataSource dataSourceRsgbBgt = ConfigUtil.getDataSourceRsgbBgt();
-            DataSource dataSourceTopNL = ConfigUtil.getDataSourceTopNL();
+            DataSource dataSourceRsgbBgt = null;
+            DataSource dataSourceTopNL = null;
+            if (LaadprocesTransformatieProces.LaadprocesSoorten.BR_TOPNL.getSoort().equalsIgnoreCase(this.config.getSoort())) {
+                dataSourceTopNL = ConfigUtil.getDataSourceTopNL();
+            }
+            if (LaadprocesTransformatieProces.LaadprocesSoorten.BR_BGTLIGHT.getSoort().equalsIgnoreCase(this.config.getSoort())) {
+                dataSourceRsgbBgt = ConfigUtil.getDataSourceRsgbBgt();
+            }
             brmo = new BrmoFramework(dataSourceStaging, null, dataSourceRsgbBgt, dataSourceTopNL);
             long[] lpIds = null;
             

--- a/brmo-service/src/main/webapp/WEB-INF/jsp/laadproces/list.jsp
+++ b/brmo-service/src/main/webapp/WEB-INF/jsp/laadproces/list.jsp
@@ -11,7 +11,7 @@
         <h1>Overzicht Laadprocessen</h1>
         <p><b>Shift-click:</b> Selecteer meerdere records,
             <b>Ctrl-click:</b> De-selecteer record.
-            <b>Let op:</b> Als U "bgtlight" wilt transformeren mogen alleen processen van deze soort geselecteerd worden.
+            <b>Let op:</b> Als U "bgtlight" of "topnl" wilt transformeren mogen alleen processen van deze een enkele soort geselecteerd worden.
         </p>
         <div id="comment-div"></div>
         <div id="laadproces-grid" class="grid-container"></div>

